README.md

```diff
@@ -147,6 +166,48 @@ This example will configure the PostgeSQL check through **autodiscovery**:
 ```
 
 
+Pinning an integration
```

Comment:
Nit: Pinning -> Upgrading/Installing? You can't use this to pin the version that comes with the agent, or an older one, so I wouldn't say it's "pinning".